### PR TITLE
Manually translated version of the tutorial (Simplified Chinese)

### PR DESCRIPTION
**Pull request by HarrisonQi**
After 3 months, 2 translated versions, I finally completed the simplified Chinese version of the Flame basic tutorial! There are quite a lot of game lovers and game developers in China, and I hope that Flame can carry forward in China with good translation!

### CODE REVIEW

**Comment by HarrisonQi** [comment]
I will gradually change the code highlighting theme to prism.js to improve the aesthetics😃. Now the [first tutorial](https://www.bugcatt.com/archives/279) has been replaced.

File: readme.md [hunk 1]
```diff
@@ -30,6 +30,15 @@
 - [Scoring, Storage, and Sound Tutorial - Step by Step with Flame and Flutter (Part 4 of 5)](https://jap.alekhin.io/scoring-storage-sound-tutorial-flame-flutter-part-4)
 - [Game Finishing and Packaging Tutorial - Step by Step with Flame and Flutter (Part 5 of 5)](https://jap.alekhin.io/game-finishing-packaging-tutorial-flame-flutter-part-5)
 
+### 简体中文教程 (Chinese translate) by [HarrisonQI](https://github.com/HarrisonQi)
```
Comment (impulse) [inline readme.md:33]: 因为你的文章是翻译的，我们可能应该参考原始的文章
比如说：简体中文教程 Alekhin Tutorial Series (Simplified Chinese) by [HarrisonQI](https://github.com/HarrisonQi)

File: readme.md [hunk 2]
```diff
@@ -30,6 +30,15 @@
...
+Because of Chinese government policy, it may be slower if you are outside of China. But the speed in China is very fast.
```
Comment (impulse) [inline readme.md:34]: 我觉得你可以删掉这个信息，因为多数看这篇文章的人来自大陆。

Comment (HarrisonQi) [reply readme.md]: Okay

File: readme.md (hunk 1 above)
Comment (HarrisonQi) [reply readme.md]: Okay

**Comment by HarrisonQi** [comment]
The above two issues have been fixed😃😃

File: readme.md (hunk 1 above)
Comment (HarrisonQi) [reply readme.md]: Flame is awsome!

**Comment by impulse** [comment]
Merging this then 👍

**Pull request merged**